packages/evolution-frontend/src/components/inputs/InputMapFindPlace.tsx

```diff
@@ -167,13 +167,16 @@ export class InputMapPoint<CustomSurvey, CustomHousehold, CustomHome, CustomPers
     };
 
     geocodePlaces = async (bbox?: [number, number, number, number]) => {
+        const language = this.props.i18n.language === 'fr' ? 'fr-CA' : 'en-CA';
```

Comment:
The geocoder only supports 'fr-CA' type for language, simply 'fr' will not work? Is there a way to get such a value from the i18n object? This PR only supports french and english, but evolution in theory supports any supported language. If you don't know how to retrieve this value, please add a FIXME in the code here and open an issue to track this.